tests/test_event_bus.c

```
@@ -0,0 +1,489 @@
+
+static void reset_state(void)
+{
+    memset(g_call_order, 0, sizeof(g_call_order));
+    g_call_count = 0;
+    g_sender_slot_received = 0;
+    g_data_received = NULL;
+    g_recursive_budget = 0;
+    obc_event_bus_init();
+}
+
+static void make_event_name(char *out, size_t chars, char fill)
```

Clang (14.0.6)

[warning] 2 adjacent parameters of 'make_event_name' of convertible types are easily swapped by mistake

(bugprone-easily-swappable-parameters)